lib/common/utils.js

```diff
@@ -287,6 +287,16 @@ exports.createRoutes = function createRoutes(files, srcDir, pagesDir) {
         }
       }
     })
+
+    if (strictRoutes || caseSensitive) {
+      route.pathToRegexpOptions = { strict: strictRoutes, sensitive: caseSensitive }
+
+      // Add final trailing slash to path on index.vue files inside pages subfolders
+      if (strictRoutes && /-index$/.test(route.name)) {
```

Comment:
How about adding a option for this behavior for not affecting legacy functionality.

Reply:
Yes you are right, I add routeCreationOptions.autoTrailingSlashSubfolders for not affect legacy funcionality